### ISSUE TITLE
Support ordering by win% (and other columns?)

### ISSUE THREAD

**Issue by mataamad** [afrojun/improved-hots-gm-leaderboard#1]
It'd be awesome to be able to order the list by winrate, or by other columns - because the default ordering doesn't really tell you much but who is best at grinding games while maintaining a reasonable winrate.

**Comment by afrojun** [comment]
Agreed, I have some free time now so will work on adding this. Thanks for the suggestion!

**Comment by afrojun** [comment]
I've got a rough version of this working now, if you get a chance try it out and let me know how it works for you. :)

**Comment by mataamad** [comment]
Awesome, much more useful! A few things I noticed:
- It seems to bug out if I 'load more' - it sorts ascending and then refuses to sort if I click again
- Sorting by the win/loss column doesn't work right (I think)
- It'd be cool to add some sort of hint to in the UI that the columns are sortable

Thanks for the change :)
